INFO - 2024-05-16 06:29:04 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-05-16 06:30:24 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-05-16 06:30:42 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2024-05-16 06:30:42 --> mysqli_sql_exception: Table 'db_perpustakaan.users' doesn't exist in C:\xampp\htdocs\Book-Nest\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\Book-Nest\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT 1\nFROM `...', 0)
#1 C:\xampp\htdocs\Book-Nest\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1\nFROM `...')
#2 C:\xampp\htdocs\Book-Nest\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1\nFROM `...')
#3 C:\xampp\htdocs\Book-Nest\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#4 C:\xampp\htdocs\Book-Nest\system\Validation\StrictRules\Rules.php(227): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\Book-Nest\system\Validation\Validation.php(328): CodeIgniter\Validation\StrictRules\Rules->is_unique('ma6239554@gmail...', 'email', Array, NULL, 'email')
#6 C:\xampp\htdocs\Book-Nest\system\Validation\Validation.php(200): CodeIgniter\Validation\Validation->processRules('email', 'email', 'ma6239554@gmail...', Array, Array)
#7 C:\xampp\htdocs\Book-Nest\system\Controller.php(152): CodeIgniter\Validation\Validation->run()
#8 C:\xampp\htdocs\Book-Nest\app\Controllers\AuthController.php(72): CodeIgniter\Controller->validate(Array)
#9 C:\xampp\htdocs\Book-Nest\system\CodeIgniter.php(941): App\Controllers\AuthController->register()
#10 C:\xampp\htdocs\Book-Nest\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#11 C:\xampp\htdocs\Book-Nest\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\Book-Nest\public\index.php(79): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\Book-Nest\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2024-05-16 06:30:42 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'db_perpustakaan.users' doesn't exist
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `users`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 2 SYSTEMPATH\Validation\StrictRules\Rules.php(227): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\Validation\Validation.php(328): CodeIgniter\Validation\StrictRules\Rules->is_unique('ma6239554@gmail.com', 'email', [...], null, 'email')
 4 SYSTEMPATH\Validation\Validation.php(200): CodeIgniter\Validation\Validation->processRules('email', 'email', 'ma6239554@gmail.com', [...], [...])
 5 SYSTEMPATH\Controller.php(152): CodeIgniter\Validation\Validation->run()
 6 APPPATH\Controllers\AuthController.php(72): CodeIgniter\Controller->validate([...])
 7 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\AuthController->register()
 8 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
 9 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\Book-Nest\\public\\index.php')
CRITICAL - 2024-05-16 06:30:42 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'db_perpustakaan.users' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1
FROM `users`
WHERE `email` = \'ma6239554@gmail.com\'
 LIMIT 1')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1
FROM `users`
WHERE `email` = \'ma6239554@gmail.com\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `users`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 4 SYSTEMPATH\Validation\StrictRules\Rules.php(227): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\Validation\Validation.php(328): CodeIgniter\Validation\StrictRules\Rules->is_unique('ma6239554@gmail.com', 'email', [...], null, 'email')
 6 SYSTEMPATH\Validation\Validation.php(200): CodeIgniter\Validation\Validation->processRules('email', 'email', 'ma6239554@gmail.com', [...], [...])
 7 SYSTEMPATH\Controller.php(152): CodeIgniter\Validation\Validation->run()
 8 APPPATH\Controllers\AuthController.php(72): CodeIgniter\Controller->validate([...])
 9 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\AuthController->register()
10 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
11 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\Book-Nest\\public\\index.php')
CRITICAL - 2024-05-16 06:30:42 --> [Caused by] mysqli_sql_exception: Table 'db_perpustakaan.users' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('SELECT 1
FROM `users`
WHERE `email` = \'ma6239554@gmail.com\'
 LIMIT 1', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1
FROM `users`
WHERE `email` = \'ma6239554@gmail.com\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1
FROM `users`
WHERE `email` = \'ma6239554@gmail.com\'
 LIMIT 1')
 4 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `users`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 5 SYSTEMPATH\Validation\StrictRules\Rules.php(227): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\Validation\Validation.php(328): CodeIgniter\Validation\StrictRules\Rules->is_unique('ma6239554@gmail.com', 'email', [...], null, 'email')
 7 SYSTEMPATH\Validation\Validation.php(200): CodeIgniter\Validation\Validation->processRules('email', 'email', 'ma6239554@gmail.com', [...], [...])
 8 SYSTEMPATH\Controller.php(152): CodeIgniter\Validation\Validation->run()
 9 APPPATH\Controllers\AuthController.php(72): CodeIgniter\Controller->validate([...])
10 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\AuthController->register()
11 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
12 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\Book-Nest\\public\\index.php')
INFO - 2024-05-16 06:32:51 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-05-16 06:32:58 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2024-05-16 06:32:58 --> mysqli_sql_exception: Table 'db_perpustakaan.users' doesn't exist in C:\xampp\htdocs\Book-Nest\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\Book-Nest\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT 1\nFROM `...', 0)
#1 C:\xampp\htdocs\Book-Nest\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1\nFROM `...')
#2 C:\xampp\htdocs\Book-Nest\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1\nFROM `...')
#3 C:\xampp\htdocs\Book-Nest\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#4 C:\xampp\htdocs\Book-Nest\system\Validation\StrictRules\Rules.php(227): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\Book-Nest\system\Validation\Validation.php(328): CodeIgniter\Validation\StrictRules\Rules->is_unique('ma6239554@gmail...', 'email', Array, NULL, 'email')
#6 C:\xampp\htdocs\Book-Nest\system\Validation\Validation.php(200): CodeIgniter\Validation\Validation->processRules('email', 'email', 'ma6239554@gmail...', Array, Array)
#7 C:\xampp\htdocs\Book-Nest\system\Controller.php(152): CodeIgniter\Validation\Validation->run()
#8 C:\xampp\htdocs\Book-Nest\app\Controllers\AuthController.php(72): CodeIgniter\Controller->validate(Array)
#9 C:\xampp\htdocs\Book-Nest\system\CodeIgniter.php(941): App\Controllers\AuthController->register()
#10 C:\xampp\htdocs\Book-Nest\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#11 C:\xampp\htdocs\Book-Nest\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\Book-Nest\public\index.php(79): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\Book-Nest\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2024-05-16 06:32:58 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'db_perpustakaan.users' doesn't exist
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `users`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 2 SYSTEMPATH\Validation\StrictRules\Rules.php(227): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\Validation\Validation.php(328): CodeIgniter\Validation\StrictRules\Rules->is_unique('ma6239554@gmail.com', 'email', [...], null, 'email')
 4 SYSTEMPATH\Validation\Validation.php(200): CodeIgniter\Validation\Validation->processRules('email', 'email', 'ma6239554@gmail.com', [...], [...])
 5 SYSTEMPATH\Controller.php(152): CodeIgniter\Validation\Validation->run()
 6 APPPATH\Controllers\AuthController.php(72): CodeIgniter\Controller->validate([...])
 7 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\AuthController->register()
 8 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
 9 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\Book-Nest\\public\\index.php')
CRITICAL - 2024-05-16 06:32:58 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'db_perpustakaan.users' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1
FROM `users`
WHERE `email` = \'ma6239554@gmail.com\'
 LIMIT 1')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1
FROM `users`
WHERE `email` = \'ma6239554@gmail.com\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `users`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 4 SYSTEMPATH\Validation\StrictRules\Rules.php(227): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\Validation\Validation.php(328): CodeIgniter\Validation\StrictRules\Rules->is_unique('ma6239554@gmail.com', 'email', [...], null, 'email')
 6 SYSTEMPATH\Validation\Validation.php(200): CodeIgniter\Validation\Validation->processRules('email', 'email', 'ma6239554@gmail.com', [...], [...])
 7 SYSTEMPATH\Controller.php(152): CodeIgniter\Validation\Validation->run()
 8 APPPATH\Controllers\AuthController.php(72): CodeIgniter\Controller->validate([...])
 9 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\AuthController->register()
10 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
11 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\Book-Nest\\public\\index.php')
CRITICAL - 2024-05-16 06:32:58 --> [Caused by] mysqli_sql_exception: Table 'db_perpustakaan.users' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('SELECT 1
FROM `users`
WHERE `email` = \'ma6239554@gmail.com\'
 LIMIT 1', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1
FROM `users`
WHERE `email` = \'ma6239554@gmail.com\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1
FROM `users`
WHERE `email` = \'ma6239554@gmail.com\'
 LIMIT 1')
 4 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `users`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 5 SYSTEMPATH\Validation\StrictRules\Rules.php(227): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\Validation\Validation.php(328): CodeIgniter\Validation\StrictRules\Rules->is_unique('ma6239554@gmail.com', 'email', [...], null, 'email')
 7 SYSTEMPATH\Validation\Validation.php(200): CodeIgniter\Validation\Validation->processRules('email', 'email', 'ma6239554@gmail.com', [...], [...])
 8 SYSTEMPATH\Controller.php(152): CodeIgniter\Validation\Validation->run()
 9 APPPATH\Controllers\AuthController.php(72): CodeIgniter\Controller->validate([...])
10 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\AuthController->register()
11 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
12 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\Book-Nest\\public\\index.php')
INFO - 2024-05-16 06:37:49 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-05-16 06:38:14 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-05-16 06:38:30 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-05-16 06:42:41 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2024-05-16 06:42:41 --> mysqli_sql_exception: Table 'db_perpustakaan.users' doesn't exist in C:\xampp\htdocs\Book-Nest\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\Book-Nest\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT 1\nFROM `...', 0)
#1 C:\xampp\htdocs\Book-Nest\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1\nFROM `...')
#2 C:\xampp\htdocs\Book-Nest\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1\nFROM `...')
#3 C:\xampp\htdocs\Book-Nest\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#4 C:\xampp\htdocs\Book-Nest\system\Validation\StrictRules\Rules.php(227): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\Book-Nest\system\Validation\Validation.php(328): CodeIgniter\Validation\StrictRules\Rules->is_unique('ma6239554@gmail...', 'email', Array, NULL, 'email')
#6 C:\xampp\htdocs\Book-Nest\system\Validation\Validation.php(200): CodeIgniter\Validation\Validation->processRules('email', 'email', 'ma6239554@gmail...', Array, Array)
#7 C:\xampp\htdocs\Book-Nest\system\Controller.php(152): CodeIgniter\Validation\Validation->run()
#8 C:\xampp\htdocs\Book-Nest\app\Controllers\AuthController.php(72): CodeIgniter\Controller->validate(Array)
#9 C:\xampp\htdocs\Book-Nest\system\CodeIgniter.php(941): App\Controllers\AuthController->register()
#10 C:\xampp\htdocs\Book-Nest\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#11 C:\xampp\htdocs\Book-Nest\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\Book-Nest\public\index.php(79): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\Book-Nest\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2024-05-16 06:42:41 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'db_perpustakaan.users' doesn't exist
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `users`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 2 SYSTEMPATH\Validation\StrictRules\Rules.php(227): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\Validation\Validation.php(328): CodeIgniter\Validation\StrictRules\Rules->is_unique('ma6239554@gmail.com', 'email', [...], null, 'email')
 4 SYSTEMPATH\Validation\Validation.php(200): CodeIgniter\Validation\Validation->processRules('email', 'email', 'ma6239554@gmail.com', [...], [...])
 5 SYSTEMPATH\Controller.php(152): CodeIgniter\Validation\Validation->run()
 6 APPPATH\Controllers\AuthController.php(72): CodeIgniter\Controller->validate([...])
 7 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\AuthController->register()
 8 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
 9 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\Book-Nest\\public\\index.php')
CRITICAL - 2024-05-16 06:42:41 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'db_perpustakaan.users' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1
FROM `users`
WHERE `email` = \'ma6239554@gmail.com\'
 LIMIT 1')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1
FROM `users`
WHERE `email` = \'ma6239554@gmail.com\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `users`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 4 SYSTEMPATH\Validation\StrictRules\Rules.php(227): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\Validation\Validation.php(328): CodeIgniter\Validation\StrictRules\Rules->is_unique('ma6239554@gmail.com', 'email', [...], null, 'email')
 6 SYSTEMPATH\Validation\Validation.php(200): CodeIgniter\Validation\Validation->processRules('email', 'email', 'ma6239554@gmail.com', [...], [...])
 7 SYSTEMPATH\Controller.php(152): CodeIgniter\Validation\Validation->run()
 8 APPPATH\Controllers\AuthController.php(72): CodeIgniter\Controller->validate([...])
 9 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\AuthController->register()
10 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
11 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\Book-Nest\\public\\index.php')
CRITICAL - 2024-05-16 06:42:41 --> [Caused by] mysqli_sql_exception: Table 'db_perpustakaan.users' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('SELECT 1
FROM `users`
WHERE `email` = \'ma6239554@gmail.com\'
 LIMIT 1', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1
FROM `users`
WHERE `email` = \'ma6239554@gmail.com\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1
FROM `users`
WHERE `email` = \'ma6239554@gmail.com\'
 LIMIT 1')
 4 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `users`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 5 SYSTEMPATH\Validation\StrictRules\Rules.php(227): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\Validation\Validation.php(328): CodeIgniter\Validation\StrictRules\Rules->is_unique('ma6239554@gmail.com', 'email', [...], null, 'email')
 7 SYSTEMPATH\Validation\Validation.php(200): CodeIgniter\Validation\Validation->processRules('email', 'email', 'ma6239554@gmail.com', [...], [...])
 8 SYSTEMPATH\Controller.php(152): CodeIgniter\Validation\Validation->run()
 9 APPPATH\Controllers\AuthController.php(72): CodeIgniter\Controller->validate([...])
10 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\AuthController->register()
11 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
12 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\Book-Nest\\public\\index.php')
INFO - 2024-05-16 06:45:42 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-05-16 06:45:44 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-05-16 06:45:50 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2024-05-16 06:45:50 --> mysqli_sql_exception: Table 'db_perpustakaan.users' doesn't exist in C:\xampp\htdocs\Book-Nest\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\Book-Nest\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT 1\nFROM `...', 0)
#1 C:\xampp\htdocs\Book-Nest\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1\nFROM `...')
#2 C:\xampp\htdocs\Book-Nest\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1\nFROM `...')
#3 C:\xampp\htdocs\Book-Nest\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#4 C:\xampp\htdocs\Book-Nest\system\Validation\StrictRules\Rules.php(227): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\Book-Nest\system\Validation\Validation.php(328): CodeIgniter\Validation\StrictRules\Rules->is_unique('ma6239554@gmail...', 'email', Array, NULL, 'email')
#6 C:\xampp\htdocs\Book-Nest\system\Validation\Validation.php(200): CodeIgniter\Validation\Validation->processRules('email', 'email', 'ma6239554@gmail...', Array, Array)
#7 C:\xampp\htdocs\Book-Nest\system\Controller.php(152): CodeIgniter\Validation\Validation->run()
#8 C:\xampp\htdocs\Book-Nest\app\Controllers\AuthController.php(72): CodeIgniter\Controller->validate(Array)
#9 C:\xampp\htdocs\Book-Nest\system\CodeIgniter.php(941): App\Controllers\AuthController->register()
#10 C:\xampp\htdocs\Book-Nest\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#11 C:\xampp\htdocs\Book-Nest\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\Book-Nest\public\index.php(79): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\Book-Nest\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2024-05-16 06:45:50 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'db_perpustakaan.users' doesn't exist
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `users`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 2 SYSTEMPATH\Validation\StrictRules\Rules.php(227): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\Validation\Validation.php(328): CodeIgniter\Validation\StrictRules\Rules->is_unique('ma6239554@gmail.com', 'email', [...], null, 'email')
 4 SYSTEMPATH\Validation\Validation.php(200): CodeIgniter\Validation\Validation->processRules('email', 'email', 'ma6239554@gmail.com', [...], [...])
 5 SYSTEMPATH\Controller.php(152): CodeIgniter\Validation\Validation->run()
 6 APPPATH\Controllers\AuthController.php(72): CodeIgniter\Controller->validate([...])
 7 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\AuthController->register()
 8 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
 9 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\Book-Nest\\public\\index.php')
CRITICAL - 2024-05-16 06:45:50 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'db_perpustakaan.users' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1
FROM `users`
WHERE `email` = \'ma6239554@gmail.com\'
 LIMIT 1')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1
FROM `users`
WHERE `email` = \'ma6239554@gmail.com\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `users`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 4 SYSTEMPATH\Validation\StrictRules\Rules.php(227): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\Validation\Validation.php(328): CodeIgniter\Validation\StrictRules\Rules->is_unique('ma6239554@gmail.com', 'email', [...], null, 'email')
 6 SYSTEMPATH\Validation\Validation.php(200): CodeIgniter\Validation\Validation->processRules('email', 'email', 'ma6239554@gmail.com', [...], [...])
 7 SYSTEMPATH\Controller.php(152): CodeIgniter\Validation\Validation->run()
 8 APPPATH\Controllers\AuthController.php(72): CodeIgniter\Controller->validate([...])
 9 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\AuthController->register()
10 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
11 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\Book-Nest\\public\\index.php')
CRITICAL - 2024-05-16 06:45:50 --> [Caused by] mysqli_sql_exception: Table 'db_perpustakaan.users' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('SELECT 1
FROM `users`
WHERE `email` = \'ma6239554@gmail.com\'
 LIMIT 1', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1
FROM `users`
WHERE `email` = \'ma6239554@gmail.com\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1
FROM `users`
WHERE `email` = \'ma6239554@gmail.com\'
 LIMIT 1')
 4 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `users`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 5 SYSTEMPATH\Validation\StrictRules\Rules.php(227): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\Validation\Validation.php(328): CodeIgniter\Validation\StrictRules\Rules->is_unique('ma6239554@gmail.com', 'email', [...], null, 'email')
 7 SYSTEMPATH\Validation\Validation.php(200): CodeIgniter\Validation\Validation->processRules('email', 'email', 'ma6239554@gmail.com', [...], [...])
 8 SYSTEMPATH\Controller.php(152): CodeIgniter\Validation\Validation->run()
 9 APPPATH\Controllers\AuthController.php(72): CodeIgniter\Controller->validate([...])
10 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\AuthController->register()
11 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
12 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\Book-Nest\\public\\index.php')
CRITICAL - 2024-05-16 07:40:24 --> Error: Cannot call constructor
in APPPATH\Controllers\InputController.php on line 11.
 1 SYSTEMPATH\CodeIgniter.php(913): App\Controllers\InputController->__construct()
 2 SYSTEMPATH\CodeIgniter.php(493): CodeIgniter\CodeIgniter->createController()
 3 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\Book-Nest\\public\\index.php')
CRITICAL - 2024-05-16 07:43:44 --> Error: Cannot call constructor
in APPPATH\Controllers\InputController.php on line 11.
 1 SYSTEMPATH\CodeIgniter.php(913): App\Controllers\InputController->__construct()
 2 SYSTEMPATH\CodeIgniter.php(493): CodeIgniter\CodeIgniter->createController()
 3 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\Book-Nest\\public\\index.php')
CRITICAL - 2024-05-16 07:44:56 --> Error: Cannot call constructor
in APPPATH\Controllers\InputController.php on line 11.
 1 SYSTEMPATH\CodeIgniter.php(913): App\Controllers\InputController->__construct()
 2 SYSTEMPATH\CodeIgniter.php(493): CodeIgniter\CodeIgniter->createController()
 3 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\Book-Nest\\public\\index.php')
CRITICAL - 2024-05-16 07:44:58 --> Error: Cannot call constructor
in APPPATH\Controllers\InputController.php on line 11.
 1 SYSTEMPATH\CodeIgniter.php(913): App\Controllers\InputController->__construct()
 2 SYSTEMPATH\CodeIgniter.php(493): CodeIgniter\CodeIgniter->createController()
 3 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\Book-Nest\\public\\index.php')
CRITICAL - 2024-05-16 07:46:17 --> Error: Cannot call constructor
in APPPATH\Controllers\InputController.php on line 13.
 1 SYSTEMPATH\CodeIgniter.php(913): App\Controllers\InputController->__construct()
 2 SYSTEMPATH\CodeIgniter.php(493): CodeIgniter\CodeIgniter->createController()
 3 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\Book-Nest\\public\\index.php')
CRITICAL - 2024-05-16 07:46:21 --> Error: Cannot call constructor
in APPPATH\Controllers\InputController.php on line 13.
 1 SYSTEMPATH\CodeIgniter.php(913): App\Controllers\InputController->__construct()
 2 SYSTEMPATH\CodeIgniter.php(493): CodeIgniter\CodeIgniter->createController()
 3 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\Book-Nest\\public\\index.php')
ERROR - 2024-05-16 07:48:25 --> mysqli_sql_exception: Table 'db_perpustakaan.bukus' doesn't exist in C:\xampp\htdocs\Book-Nest\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\Book-Nest\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `bu...', 0)
#1 C:\xampp\htdocs\Book-Nest\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `bu...')
#2 C:\xampp\htdocs\Book-Nest\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `bu...')
#3 C:\xampp\htdocs\Book-Nest\system\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `bu...', Array, false)
#4 C:\xampp\htdocs\Book-Nest\system\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\Book-Nest\system\BaseModel.php(802): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\Book-Nest\system\Model.php(744): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\Book-Nest\app\Models\Buku.php(51): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\Book-Nest\app\Controllers\InputController.php(22): App\Models\Buku->save_data('testing', '2009', '1', '1')
#9 C:\xampp\htdocs\Book-Nest\system\CodeIgniter.php(941): App\Controllers\InputController->submit()
#10 C:\xampp\htdocs\Book-Nest\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\InputController))
#11 C:\xampp\htdocs\Book-Nest\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\Book-Nest\public\index.php(79): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\Book-Nest\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2024-05-16 07:48:25 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'db_perpustakaan.bukus' doesn't exist
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `bukus` (`judul`, `tahun`, `jumlah`, `loker`) VALUES (:judul:, :tahun:, :jumlah:, :loker:)', [...], false)
 2 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Models\Buku.php(51): CodeIgniter\Model->insert([...])
 6 APPPATH\Controllers\InputController.php(22): App\Models\Buku->save_data('testing', '2009', '1', '1')
 7 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\InputController->submit()
 8 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\InputController))
 9 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\Book-Nest\\public\\index.php')
CRITICAL - 2024-05-16 07:48:25 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'db_perpustakaan.bukus' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `bukus` (`judul`, `tahun`, `jumlah`, `loker`) VALUES (\'testing\', \'2009\', \'1\', \'1\')')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `bukus` (`judul`, `tahun`, `jumlah`, `loker`) VALUES (\'testing\', \'2009\', \'1\', \'1\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `bukus` (`judul`, `tahun`, `jumlah`, `loker`) VALUES (:judul:, :tahun:, :jumlah:, :loker:)', [...], false)
 4 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Models\Buku.php(51): CodeIgniter\Model->insert([...])
 8 APPPATH\Controllers\InputController.php(22): App\Models\Buku->save_data('testing', '2009', '1', '1')
 9 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\InputController->submit()
10 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\InputController))
11 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\Book-Nest\\public\\index.php')
CRITICAL - 2024-05-16 07:48:25 --> [Caused by] mysqli_sql_exception: Table 'db_perpustakaan.bukus' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `bukus` (`judul`, `tahun`, `jumlah`, `loker`) VALUES (\'testing\', \'2009\', \'1\', \'1\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `bukus` (`judul`, `tahun`, `jumlah`, `loker`) VALUES (\'testing\', \'2009\', \'1\', \'1\')')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `bukus` (`judul`, `tahun`, `jumlah`, `loker`) VALUES (\'testing\', \'2009\', \'1\', \'1\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `bukus` (`judul`, `tahun`, `jumlah`, `loker`) VALUES (:judul:, :tahun:, :jumlah:, :loker:)', [...], false)
 5 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Models\Buku.php(51): CodeIgniter\Model->insert([...])
 9 APPPATH\Controllers\InputController.php(22): App\Models\Buku->save_data('testing', '2009', '1', '1')
10 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\InputController->submit()
11 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\InputController))
12 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\Book-Nest\\public\\index.php')
ERROR - 2024-05-16 07:49:09 --> mysqli_sql_exception: Unknown column 'loker' in 'field list' in C:\xampp\htdocs\Book-Nest\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\Book-Nest\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `tb...', 0)
#1 C:\xampp\htdocs\Book-Nest\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tb...')
#2 C:\xampp\htdocs\Book-Nest\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tb...')
#3 C:\xampp\htdocs\Book-Nest\system\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tb...', Array, false)
#4 C:\xampp\htdocs\Book-Nest\system\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\Book-Nest\system\BaseModel.php(802): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\Book-Nest\system\Model.php(744): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\Book-Nest\app\Models\Buku.php(51): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\Book-Nest\app\Controllers\InputController.php(22): App\Models\Buku->save_data('tes', '', '', '1')
#9 C:\xampp\htdocs\Book-Nest\system\CodeIgniter.php(941): App\Controllers\InputController->submit()
#10 C:\xampp\htdocs\Book-Nest\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\InputController))
#11 C:\xampp\htdocs\Book-Nest\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\Book-Nest\public\index.php(79): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\Book-Nest\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2024-05-16 07:49:09 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'loker' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tb_buku` (`judul`, `tahun`, `jumlah`, `loker`) VALUES (:judul:, :tahun:, :jumlah:, :loker:)', [...], false)
 2 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Models\Buku.php(51): CodeIgniter\Model->insert([...])
 6 APPPATH\Controllers\InputController.php(22): App\Models\Buku->save_data('tes', '', '', '1')
 7 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\InputController->submit()
 8 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\InputController))
 9 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\Book-Nest\\public\\index.php')
CRITICAL - 2024-05-16 07:49:09 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'loker' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tb_buku` (`judul`, `tahun`, `jumlah`, `loker`) VALUES (\'tes\', \'\', \'\', \'1\')')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tb_buku` (`judul`, `tahun`, `jumlah`, `loker`) VALUES (\'tes\', \'\', \'\', \'1\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tb_buku` (`judul`, `tahun`, `jumlah`, `loker`) VALUES (:judul:, :tahun:, :jumlah:, :loker:)', [...], false)
 4 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Models\Buku.php(51): CodeIgniter\Model->insert([...])
 8 APPPATH\Controllers\InputController.php(22): App\Models\Buku->save_data('tes', '', '', '1')
 9 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\InputController->submit()
10 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\InputController))
11 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\Book-Nest\\public\\index.php')
CRITICAL - 2024-05-16 07:49:09 --> [Caused by] mysqli_sql_exception: Unknown column 'loker' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `tb_buku` (`judul`, `tahun`, `jumlah`, `loker`) VALUES (\'tes\', \'\', \'\', \'1\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tb_buku` (`judul`, `tahun`, `jumlah`, `loker`) VALUES (\'tes\', \'\', \'\', \'1\')')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tb_buku` (`judul`, `tahun`, `jumlah`, `loker`) VALUES (\'tes\', \'\', \'\', \'1\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tb_buku` (`judul`, `tahun`, `jumlah`, `loker`) VALUES (:judul:, :tahun:, :jumlah:, :loker:)', [...], false)
 5 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Models\Buku.php(51): CodeIgniter\Model->insert([...])
 9 APPPATH\Controllers\InputController.php(22): App\Models\Buku->save_data('tes', '', '', '1')
10 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\InputController->submit()
11 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\InputController))
12 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\Book-Nest\\public\\index.php')
ERROR - 2024-05-16 07:49:20 --> mysqli_sql_exception: Unknown column 'loker' in 'field list' in C:\xampp\htdocs\Book-Nest\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\Book-Nest\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `tb...', 0)
#1 C:\xampp\htdocs\Book-Nest\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tb...')
#2 C:\xampp\htdocs\Book-Nest\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tb...')
#3 C:\xampp\htdocs\Book-Nest\system\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tb...', Array, false)
#4 C:\xampp\htdocs\Book-Nest\system\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\Book-Nest\system\BaseModel.php(802): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\Book-Nest\system\Model.php(744): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\Book-Nest\app\Models\Buku.php(51): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\Book-Nest\app\Controllers\InputController.php(22): App\Models\Buku->save_data('tes', '2009', '1', '1')
#9 C:\xampp\htdocs\Book-Nest\system\CodeIgniter.php(941): App\Controllers\InputController->submit()
#10 C:\xampp\htdocs\Book-Nest\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\InputController))
#11 C:\xampp\htdocs\Book-Nest\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\Book-Nest\public\index.php(79): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\Book-Nest\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2024-05-16 07:49:20 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'loker' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tb_buku` (`judul`, `tahun`, `jumlah`, `loker`) VALUES (:judul:, :tahun:, :jumlah:, :loker:)', [...], false)
 2 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Models\Buku.php(51): CodeIgniter\Model->insert([...])
 6 APPPATH\Controllers\InputController.php(22): App\Models\Buku->save_data('tes', '2009', '1', '1')
 7 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\InputController->submit()
 8 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\InputController))
 9 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\Book-Nest\\public\\index.php')
CRITICAL - 2024-05-16 07:49:20 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'loker' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tb_buku` (`judul`, `tahun`, `jumlah`, `loker`) VALUES (\'tes\', \'2009\', \'1\', \'1\')')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tb_buku` (`judul`, `tahun`, `jumlah`, `loker`) VALUES (\'tes\', \'2009\', \'1\', \'1\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tb_buku` (`judul`, `tahun`, `jumlah`, `loker`) VALUES (:judul:, :tahun:, :jumlah:, :loker:)', [...], false)
 4 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Models\Buku.php(51): CodeIgniter\Model->insert([...])
 8 APPPATH\Controllers\InputController.php(22): App\Models\Buku->save_data('tes', '2009', '1', '1')
 9 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\InputController->submit()
10 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\InputController))
11 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\Book-Nest\\public\\index.php')
CRITICAL - 2024-05-16 07:49:20 --> [Caused by] mysqli_sql_exception: Unknown column 'loker' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `tb_buku` (`judul`, `tahun`, `jumlah`, `loker`) VALUES (\'tes\', \'2009\', \'1\', \'1\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tb_buku` (`judul`, `tahun`, `jumlah`, `loker`) VALUES (\'tes\', \'2009\', \'1\', \'1\')')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tb_buku` (`judul`, `tahun`, `jumlah`, `loker`) VALUES (\'tes\', \'2009\', \'1\', \'1\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tb_buku` (`judul`, `tahun`, `jumlah`, `loker`) VALUES (:judul:, :tahun:, :jumlah:, :loker:)', [...], false)
 5 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Models\Buku.php(51): CodeIgniter\Model->insert([...])
 9 APPPATH\Controllers\InputController.php(22): App\Models\Buku->save_data('tes', '2009', '1', '1')
10 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\InputController->submit()
11 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\InputController))
12 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\Book-Nest\\public\\index.php')
